Assets/HoloToolkit/Utilities/Scripts/TextToSpeechManager.cs

```diff
@@ -29,9 +56,13 @@ public class TextToSpeechManager : MonoBehaviour
         [Tooltip("The audio source where speech will be played.")]
         public AudioSource audioSource;
 
+        [Tooltip("The voice that will be used to generate speech.")]
+        public TextToSpeechVoice voice;
```

Comment:
Actually, I take that back. If we do SerializeField and wrap that field in properties we shouldn't be creating a breaking change. I'll make that change now and update.